src/lib/Tab/LocationView/UrlsTab.php

```diff
@@ -45,6 +47,9 @@ class UrlsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
     /** @var \eZ\Publish\API\Repository\PermissionResolver */
     protected $permissionResolver;
 
+    /** @var \eZ\Publish\Core\Helper\TranslationHelper */
+    protected $translationHelper;
```

Comment:
```suggestion
    private $translationHelper;
```

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1131/commits/06c47b5733b8baf5e4e121603993a69c05fd9fa7